### PR TITLE
Update Windows SDK and VC platform toolsets.

### PR DESCRIPTION
**Pull request by JunielKatarn**
This change defaults to the corresponding Visual Studio platform toolset (v141 for 2017, v142 for 2019).
It also updates the NUSPEC for better flexibility (custom ID, if needed, instead of forced to the NUSPEC base file name).

### CODE REVIEW

File: PropertySheets/Chakra.Cpp.props [hunk 1]
```diff
@@ -4,11 +4,11 @@
   <PropertyGroup Label="Globals">
     <VCProjectVersion>15.0</VCProjectVersion>
     <Keyword>Win32Proj</Keyword>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
 
   <PropertyGroup Label="Configuration">
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>$(DefaultPlatformToolset)</PlatformToolset>
```
Comment (akrantz) [inline PropertySheets/Chakra.Cpp.props]: This allows using the default compiler from VS 2017 or VS 2019 now?

File: NuGet/Microsoft.ChakraCore.Debugger.nuspec [hunk 2]
```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  Sample usage:
+  nuget.exe pack Microsoft.ChakraCore.Debugger.nuspec -Properties "id=some_id" -Version "some_version"
+-->
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.ChakraCore.Debugger</id>
+    <id>$id$</id>
```
Comment (akrantz) [inline NuGet/Microsoft.ChakraCore.Debugger.nuspec]: Does this change it to "ChakraCore.Debugger" for the NuGet id?

Comment (JunielKatarn) [reply NuGet/Microsoft.ChakraCore.Debugger.nuspec]: It allows setting the ID to whatever we find convenient.
If, for instance, in the future we deploy a NuGet called `Microsoft.ChakraDebugger`, we can do that without changing the NUSPEC.

The `id` can be injected via the `Properties` argument in the NuGet command.

File: PropertySheets/Chakra.Cpp.props (hunk 1 above)
Comment (JunielKatarn) [reply PropertySheets/Chakra.Cpp.props]: Yes. This has the following behavior:
2017: Defaults to v141.
2019: Defaults to v142.

It's always overridable, of course.

**Comment by JunielKatarn** [comment]
Rebase was very unsuccessful. Will try again in a new PR.

**Pull request closed without merging**